<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lifePreserver</class>
 <widget class="QMainWindow" name="lifePreserver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Life Preserver</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lPreserve.qrc">
    <normaloff>:/images/images/lifepreserver.png</normaloff>:/images/images/lifepreserver.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treePreservers">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>175</number>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Backup Server</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Last Backup</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Schedule</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushPreserverStart">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="icon">
         <iconset resource="lPreserve.qrc">
          <normaloff>:/images/images/system-run.png</normaloff>:/images/images/system-run.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>697</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_Preserver"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Preservers">
    <property name="title">
     <string>&amp;Preservers</string>
    </property>
    <addaction name="action_Edit"/>
    <addaction name="actionRestore"/>
    <addaction name="separator"/>
    <addaction name="action_Remove"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Preservers"/>
  </widget>
  <action name="action_New_Preserver">
   <property name="text">
    <string>&amp;New Preserver</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Edit">
   <property name="text">
    <string>&amp;Edit</string>
   </property>
  </action>
  <action name="action_Remove">
   <property name="text">
    <string>&amp;Remove</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>Restore From</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="lPreserve.qrc"/>
 </resources>
 <connections/>
</ui>
